pkg/restclient/volume.go

```diff
@@ -0,0 +1,42 @@
+package restclient
+
+import (
+	"encoding/json"
+	"fmt"
+	"strings"
+
+	"github.com/gluster/glusterd2/pkg/api"
+)
+
+// VolumeCreate creates Gluster Volume
+func (c *RESTClient) VolumeCreate(volname string, bricks []string, replica int, force bool) error {
```

Comment:
Makes sense. Let me think about using VolumeCreateOptions struct or something similar for optional parameters.